examples/async_chat_sdk_output_format/app.py

```diff
@@ -0,0 +1,40 @@
+import agenta as ag
+from agenta import FloatParam, MessagesInput, MultipleChoiceParam
+from openai import AsyncOpenAI
+
+
+client = AsyncOpenAI()
+
+SYSTEM_PROMPT = "You have expertise in offering technical ideas to startups."
+CHAT_LLM_GPT = [
+    "gpt-3.5-turbo-16k",
+    "gpt-3.5-turbo-0301",
+    "gpt-3.5-turbo-0613",
+    "gpt-3.5-turbo-16k-0613",
+    "gpt-4",
+]
+
+ag.init()
+ag.config.default(
+    temperature=FloatParam(0.2),
+    model=MultipleChoiceParam("gpt-3.5-turbo", CHAT_LLM_GPT),
+    max_tokens=ag.IntParam(-1, -1, 4000),
+    prompt_system=ag.TextParam(SYSTEM_PROMPT),
+)
+
+
+@ag.entrypoint
+async def chat(inputs: MessagesInput = MessagesInput()) -> str:
```

Comment:
let's remove the output type since it's not str anymroe

Reply:
Alright.